### ISSUE TITLE
fluentd to kafka error

### ISSUE THREAD

**Issue by D-wenhao** [fluent/fluent-plugin-kafka#385]
# plugin version

> fluent-plugin-kafka (0.7.9)
> ruby-kafka (0.5.5)

# ERROR log messenage
```
2021-01-08 04:24:01 +0000 [warn]: Send exception occurred: wrong number of arguments (given 1, expected 2)
2021-01-08 04:24:01.975802143 +0000 fluent.warn: {"message":"Send exception occurred: wrong number of arguments (given 1, expected 2)"}
2021-01-08 04:24:01 +0000 [warn]: Exception Backtrace : /usr/local/bundle/gems/ruby-kafka-0.5.5/lib/kafka.rb:220:in `initialize'
/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/kafka_producer_ext.rb:184:in `exception'
/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/kafka_producer_ext.rb:184:in `raise'
/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/kafka_producer_ext.rb:184:in `deliver_messages_with_retries'
/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/kafka_producer_ext.rb:102:in `deliver_messages'
/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/out_kafka2.rb:220:in `write'
/usr/local/bundle/gems/fluentd-1.11.1/lib/fluent/plugin/output.rb:1133:in `try_flush'
/usr/local/bundle/gems/fluentd-1.11.1/lib/fluent/plugin/output.rb:1439:in `flush_thread_run'
/usr/local/bundle/gems/fluentd-1.11.1/lib/fluent/plugin/output.rb:461:in `block (2 levels) in start'
/usr/local/bundle/gems/fluentd-1.11.1/lib/fluent/plugin_helper/thread.rb:78:in `block in thread_create'
2021-01-08 04:24:01.975977939 +0000 fluent.warn: {"message":"Exception Backtrace : /usr/local/bundle/gems/ruby-kafka-0.5.5/lib/kafka.rb:220:in `initialize'\n/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/kafka_producer_ext.rb:184:in `exception'\n/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/kafka_producer_ext.rb:184:in `raise'\n/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/kafka_producer_ext.rb:184:in `deliver_messages_with_retries'\n/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/kafka_producer_ext.rb:102:in `deliver_messages'\n/usr/local/bundle/gems/fluent-plugin-kafka-0.7.9/lib/fluent/plugin/out_kafka2.rb:220:in `write'\n/usr/local/bundle/gems/fluentd-1.11.1/lib/fluent/plugin/output.rb:1133:in `try_flush'\n/usr/local/bundle/gems/fluentd-1.11.1/lib/fluent/plugin/output.rb:1439:in `flush_thread_run'\n/usr/local/bundle/gems/fluentd-1.11.1/lib/fluent/plugin/output.rb:461:in `block (2 levels) in start'\n/usr/local/bundle/gems/fluentd-1.11.1/lib/fluent/plugin_helper/thread.rb:78:in `block in thread_create'"}
2021-01-08 04:24:01 +0000 [warn]: got unrecoverable error in primary and no secondary error_class=ArgumentError error="wrong number of arguments (given 1, expected 2)"
2021-01-08 04:24:01.976307382 +0000 fluent.warn: {"error":"#<ArgumentError: wrong number of arguments (given 1, expected 2)>","message":"got unrecoverable error in primary and no secondary error_class=ArgumentError error=\"wrong number of arguments (given 1, expected 2)\""}
  2021-01-08 04:24:01 +0000 [warn]: suppressed same stacktrace
2021-01-08 04:24:01 +0000 [warn]: bad chunk is moved to /tmp/fluent/backup/worker0/object_a12ac/5b85becb88f7b3bbbce80989358128be.log
2021-01-08 04:24:01.977002248 +0000 fluent.warn: {"message":"bad chunk is moved to /tmp/fluent/backup/worker0/object_a12ac/5b85becb88f7b3bbbce80989358128be.log"}
```

# How shoud i do to fix this problem

**Comment by wangjianxiang** [comment]
hello!!!  i has fix this problem , 
please open :  get_kafka_client_log  (bool) :default => false

**Comment by kenhys** [comment]
This is caused by incompatible version.
It is same as #229, and fixed by #230.

Please use fluent-plugin-kafka 0.8.2 or later.
